docs/_layouts/global.html

```diff
@@ -207,14 +205,79 @@ <h1 class="title">{{ page.title }}</h1>
     })
 </script>
 
+<!-- Pure JavaScript implementation of ScrollSpy -->
 <script>
-    var controller = new ScrollMagic.Controller();
+    //finds toc and automatically sorts by main list items as parents and nested list items as children
 
-    new ScrollMagic.Scene({
-        offset: 20
-    })
-    .setClassToggle(".navbar-inner", "shrunk")
-    .addTo(controller);
+    var toc = document.getElementById("toc");
+    if(toc != null) {
```

Comment:
Please add a space between the "if" and the "(". Please apply this to the other "if" and "for" statements in this javascript block.